### ISSUE TITLE
Release new version for Rails 7.1

### ISSUE THREAD

**Issue by ezekg** [MaxLap/activerecord_where_assoc#13]
The change https://github.com/MaxLap/activerecord_where_assoc/commit/47e82d224d2de376c43de071bebc5e550a565e8a is currently unreleased. Can we cut a new version for Rails 7.1 support?

Edit: sponsored on behalf of my company @keygen-sh. Thanks for the great gem!

**Comment by MaxLap** [comment]
Thanks for the heads up! Didn't notice that 7.1 was at the release candidate stage.

I just released 1.1.4.

Thanks a lot for the support! I'm glad you appreciate the gem.

**Comment by joshuacronemeyer** [comment]
Hi there, thanks for this Gem. I was getting this exception from our application tests after upgrading to rails 7.1.3.3

```
NameError: uninitialized constant ActiveRecord::NullRelation
```

whenever any code called  where_assoc_exists 

Google had 0 results for me when I searched for that exception, so I thought i'd come back here and mention it. Might help somebody down the road.  So yeah nothing to worry about here but if you're seeing that error you might consider upgrading your activerecord_where_assoc gem to the latest!
